Description: Adjust behavior of ds-identify for SRU
 To make this acceptable as a SRU we have changed ds-identify to
 act in 'report only' mode, and to only 'warn' when it cloud-init
 finds itself to be using a Ec2 Datasource on an unknown and
 non AWS platform.
Forwarded: not-needed
Author: Scott Moser <smoser@ubuntu.com>
Bug-ubuntu: http://bugs.launchpad.net/bugs/1669675
Bug-ubuntu: http://bugs.launchpad.net/bugs/1660385

--- a/tools/ds-identify
+++ b/tools/ds-identify
@@ -93,8 +93,8 @@ _DI_LOGGED=""
 DI_MAIN=${DI_MAIN:-main}
 
 DI_BLKID_OUTPUT=""
-DI_DEFAULT_POLICY="search,found=all,maybe=all,notfound=${DI_DISABLED}"
-DI_DEFAULT_POLICY_NO_DMI="search,found=all,maybe=all,notfound=${DI_ENABLED}"
+DI_DEFAULT_POLICY="report,found=all,maybe=all,notfound=${DI_ENABLED}"
+DI_DEFAULT_POLICY_NO_DMI="report,found=all,maybe=all,notfound=${DI_ENABLED}"
 DI_DMI_CHASSIS_ASSET_TAG=""
 DI_DMI_PRODUCT_NAME=""
 DI_DMI_SYS_VENDOR=""
@@ -131,7 +131,7 @@ DI_ON_FOUND=""
 DI_ON_MAYBE=""
 DI_ON_NOTFOUND=""
 
-DI_EC2_STRICT_ID_DEFAULT="true"
+DI_EC2_STRICT_ID_DEFAULT="warn"
 
 _IS_IBM_CLOUD=""
 
